public/index.html

```diff
@@ -13,7 +13,7 @@
     </script>
   </head>
 
-  <body class="cpd-theme-dark">
+  <body class="nodisplay">
```

Comment:
Could we rename this class to something like `no-theme` so that it's more apparent that it's involved in the theme code?

Reply:
what about `invisible-loading-theme` (so its also clear that this class could be the reason why its not rendering anything.)

Reply:
That's also fine by me